packages/react-components/src/components/Attachment/AttachmentDownloadCards.tsx

```diff
@@ -152,7 +152,8 @@ const getDefaultMenuActions = (
 ): AttachmentMenuAction[] => {
   let actionName = locale.openAttachment;
   // if message is sent by a Teams user, we need to use a different icon ("open")
-  if (chatMessage?.senderId?.includes('8:orgid:')) {
+  const isTeamsUser = isIdentityMicrosoftTeamsUser(_toCommunicationIdentifier(chatMessage?.senderId ?? ''));
```

Comment:
This implicitly brings in `CommunicationIdentifier` as a return type and we should probably have the underlying wrapper function in the common area handle this.
`_isIdentityMicrosoftTeamsUser(value: string?) -> bool`